error/s2n_errno.h

```diff
@@ -333,6 +340,21 @@ extern __thread const char *s2n_debug_str;
  */
 #define S2N_IFF(a, b) (!!(a) == !!(b))
 
+/**
+ * These macros are used to specify code contracts in CBMC proofs.
+ */
+#ifdef CBMC
+#    define CBMC_ASSIGNS(...) __CPROVER_assigns(__VA_ARGS__)
```

Comment:
Why have the CBMC prefix on these macros? There's not really anything specific to CBMC itself, right?

Reply:
Yep, you're right. I can remove it.

Reply:
Maybe do `CONTRACT_ASSIGNS` or something? That way we're a bit flexible on which proof engine we use.

Also: we may want a `CONTRACT_RETURN_VALUE` macro as well.

Reply:
Implemented as suggested. Could you take another look @camshaft ?